include/boost/numeric/interval/hw_rounding.hpp

```diff
@@ -44,23 +46,51 @@ namespace boost {
 namespace numeric {
 namespace interval_lib {
 
+namespace detail
+{
+  template<class T>
+  struct runtime_checker
+  {
+  protected:
+    runtime_checker()
+    {
+      static helper _;
+    }
+
+  private:
+    struct helper
+    {
+      helper()
+      {
+        save_state<rounded_arith_opp<T> > rounding;
```

Comment:
Given state was saved already (see new lines 80, 86, 92) is this necessary?

Reply:
You are right, it's redundant, I have removed it.